libfoundation/src/system-file-posix.cpp

```diff
@@ -364,8 +364,6 @@ __MCSFileSetContents (MCStringRef p_native_path,
 
 	if (0 != rename (*t_temp_path_sys, *t_path_sys))
 	{
-		int t_save_errno = errno;
-
```

Comment:
Regardless it's probably another case of us all needing to use the same indentation settings. FYI my Xcode is set to indent with 4 spaces and a tab width of 4 spaces. I suspect this was originally implemented with tab indents and the tab width on github is something more than 4 spaces... there was a discussion at some point of some tool that would ensure all our settings were standardised... can't remember what it was though